### ISSUE TITLE
Style player for narrow devices (mobile)

### ISSUE THREAD

**Issue by jgeurts** [minarets/website#21]

**Comment by naastyaa** [comment]
Can i work on this?

**Comment by jgeurts** [comment]
Absolutely! I [updated the README](https://github.com/minarets/website#running-the-site-locally) with instructions that should get you going locally. Please let me know if you have troubles!
